autoscheduler/autoscheduler/settings/base.py

```diff
@@ -181,6 +181,10 @@
 
 USE_TZ = True
 
+# print(os.path.join(BASE_DIR, "assets"))
```

Comment:
Thank you for moving this file. Also, I guess we could erase the commented line now